api/v1/ptpconfig_webhook.go

```diff
@@ -171,8 +171,32 @@ func (r *PtpConfig) validate() error {
 					if _, err := strconv.ParseUint(v, 10, 32); err != nil {
 						return errors.New("inSyncConditionThreshold='" + v + "' is invalid; must be an unsigned integer")
 					}
+
+				case strings.Contains(k, "clockId"):
+					// Allow explicit clockId
+					if _, err := strconv.ParseUint(v, 10, 64); err != nil {
+						if _, err := strconv.ParseUint(v, 16, 64); err != nil {
+							return errors.New("clockId='" + v + "' is invalid; must be an unsigned integer")
```

Comment:
Might want to mention this is expecting hex in the error message

Reply:
It is expecting either decimal or hex